integration/test/crmapping/crmapping_test.go

```diff
@@ -229,8 +229,8 @@ func ensureCRsExist(t *testing.T, client client.Client, inputFiles []string) {
 
 		if o.GetObjectKind().GroupVersionKind().Kind == "NamespaceList" {
 			lst := o.(*corev1.NamespaceList)
-			for _, i := range lst.Items {
-				err = client.Create(context.Background(), &i)
+			for i := range lst.Items {
```

Comment:
why do we need this change? Actually why do we need all these changes?

Reply:
I guess they are linter recommendations

Reply:
`golangci-lint` has changed with architect-orb and it's raising errors on **go-build**:
```
service/controller/cluster/handler/clusterownerreference/resource.go:187:57: G601: Implicit memory aliasing in for loop. (gosec)
		err = controllerutil.SetControllerReference(&cluster, &machinePool, r.scheme)
```
https://app.circleci.com/pipelines/github/giantswarm/azure-operator/4330/workflows/d98f096b-84a2-46e7-8c3b-dad73762642f/jobs/21294/parallel-runs/0/steps/0-116

Reply:
If there's a better way of doing this, then I'll happily change my approach.